packaging/fedora/snapd.spec

```diff
@@ -204,6 +216,10 @@ BuildRequires:  %{_bindir}/rst2man
 # ShellCheck in F24 and older doesn't work
 BuildRequires:  %{_bindir}/shellcheck
 %endif
+%if %{with apparmor}
+BuildRequires:  libapparmor-devel
+Requires:       apparmor-parser
+%endif
```

Comment:
AppArmor doesn't have a pkgconfig file? Nor is it used by the snap-confine build autofoo? This seems odd...

Reply:
It does and I meant to use it. Fixing.

Reply:
Done